specification/paths/Returns-v1-reports-count-report-entries.json

```
@@ -0,0 +1,73 @@
+{
+  "post": {
+    "tags": [
+      "ReturnReports"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "returns.manage"
+        ]
+      }
+    ],
+    "summary": "Get projected return report entries count.",
+    "description": "This endpoint retrieves the number of return report entries that would be included in a report with the supplied filters.",
+    "requestBody": {
+      "description": "The report filters.",
+      "required": true,
+      "content": {
+        "application/vnd.api+json": {
+          "schema": {
+            "type": "object",
+            "required": [
+              "data"
+            ],
+            "additionalProperties": false,
+            "properties": {
+              "data": {
+                "additionalProperties": false,
+                "properties": {
+                  "attributes": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                      "filters": {
+                        "$ref": "#/components/schemas/ReturnReportFilters"
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "responses": {
+      "201": {
```

Do we return `201` even if we don't create a record in a table somewhere?
```suggestion
      "200": {
```